src/app/modules/submission/encrypt-submission/encrypt-submission.service.ts

```diff
@@ -578,3 +580,70 @@ export const getQuarantinePresignedPostData = (
     }),
   )
 }
+
+/**
+ * Invokes lambda to scan the file in the quarantine bucket for viruses.
+ * @param quarantineFileKey object key of the file in the quarantine bucket
+ * @returns okAsync(returnPayload) if file has been successfully scanned with status 200 OK
+ * @returns errAsync(returnPayload) if lambda invocation failed or file cannot be found
+ */
+export const triggerVirusScanning = (quarantineFileKey: string) => {
+  const logMeta = {
+    action: 'triggerVirusScanning',
+    meta: {
+      quarantineFileKey,
+    },
+  }
```

Comment:
suggestion: can add a quick check here that `quarantineFileKey` is a uuid

```
import { validate } from 'uuid'
if (!validate(...))
```

Reply:
Good point, uuid check added, thanks! [34613e7](https://github.com/opengovsg/FormSG/pull/6734/commits/34613e7a263c268061a7f935236c90b5ba267d72)